narwhal/primary/src/header_waiter.rs

```diff
@@ -269,7 +276,7 @@ impl HeaderWaiter {
                         }
 
                         WaiterMessage::SyncParents(missing, header) => {
-                            debug!("Synching the parents of {header}");
+                            debug!("HeaderWaiter: Synching the parents of {header}");
```

Comment:
You don't really need the `HeaderWaiter` as this is already get printed as part of the path, example:
```
DEBUG narwhal_primary::header_waiter: Synching the parents of
```

Reply:
You are right. Will remove it next time touching this file, or just leave it to be deleted.